### DISCUSSION TITLE
[Script request] Mumble

### DISCUSSION THREAD

**Discussion by Mightyretro** [community-scripts/ProxmoxVE#460] [Request script] [answered]
### Application Name

Mumble

### Website

https://github.com/mumble-voip/mumble

### Description

Hi,

love to see this a script of this project come to life.

Kind regards

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Mati-l33t** [comment]
Why not use the Turnkey Mumble that already excist in Proxmox?

**Reply by Mightyretro** [reply]
i prefer scripts its mutch more friendly way for me and customer and friends  with same procedure as other script

**Comment by nektovse** [comment]
It would be great to make not only the mumble server itself, but also the ability to immediately install the mumble web client. Or move the mumble web client to a separate lxc container

**Comment by MickLesk** [comment, marked as answer]
https://community-scripts.org/scripts/mumble
